test/test_nn.py

```diff
@@ -11157,6 +11157,31 @@ def test_embedding_bag_device(self, device, dtype):
         self._test_EmbeddingBag(device, 'sum', True, dtype, test_backward=test_backward)
         self._test_EmbeddingBag(device, 'mean', True, dtype, test_backward=test_backward)
 
+    @dtypesIfCUDA(torch.half, torch.float, torch.double)
+    @dtypes(torch.float, torch.double)
+    def test_embedding_bag_non_contiguous_weight(self, device, dtype):
+        weight_tensor = torch.randn(4, 3, dtype=dtype, device=device)
+
+        weight_tensor_non_contig = weight_tensor[:, :3]  # This is non-contiguous strided.
```

Comment:
this is contiguous tensor

Reply:
Right! Great catch!!
Was supposed to be 
```python
weight_tensor = torch.randn(3, 4, dtype=dtype, device=device)
```

Reply:
@kshitij12345 I'll put a fix for u.

Reply:
Sure. Btw I am free this evening so I can put it up as well. Let me know if I should.

Reply:
Ah, no worries. I have a clean pytorch repo on hand, will patch this shortly.